kong/plugins/aws-lambda/api-gateway/aws/AWSBasicCredentials.lua

```diff
@@ -0,0 +1,42 @@
+--[[
+  Copyright 2016 Adobe Systems Incorporated. All rights reserved.
```

Comment:
Does the hmac and http code have to be included like this? aren't there resty modules for that?

The adobe code is Apache 2.0, haven't checked the license of the http.lua file by James Hurst, but looks like MIT.